boto/vendored/regions/exceptions.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2016 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
I don't follow how this is any simpler.  You have to curate exceptions.py to pull in the correct exceptions do you not?  You also have to do this transitively to pull in all the base class exceptions as well.  If this just a literal `cp` of a file then sure, but you already have to do manual work, I'd prefer to do the simpler option of keeping everything in a single file.